applications/CompressiblePotentialFlowApplication/custom_elements/compressible_potential_flow_element.h

```diff
@@ -52,27 +44,26 @@ namespace Kratos
 ///@name Kratos Classes
 ///@{
 
-template< int Dim, int NumNodes >
+template <int Dim, int NumNodes>
 class CompressiblePotentialFlowElement : public Element
 {
 public:
-
     template <unsigned int TNumNodes, unsigned int TDim>
     struct ElementalData
     {
-        array_1d<double,TNumNodes> phis, distances;
+        array_1d<double, TNumNodes> phis, distances;
         double vol;
 
-        bounded_matrix<double, TNumNodes, TDim > DN_DX;
-        array_1d<double, TNumNodes > N;
+        BoundedMatrix<double, TNumNodes, TDim> DN_DX;
+        array_1d<double, TNumNodes> N;
     };
 
-
-
     ///@name Type Definitions
     ///@{
 
     typedef Element BaseType;
+    static constexpr int TNumNodes = NumNodes;
```

Comment:
We do this in order to be able to access the template parameters from the adjoint element:

https://github.com/KratosMultiphysics/Kratos/blob/9de39ab24f0e6f9064727280c394fb30e3647d47/applications/CompressiblePotentialFlowApplication/custom_elements/adjoint_potential_flow_element.h#L33-L34

Tagging @marcnunezc  @armingeiser  and @MFusseder because I think they already discussed this. Is there a cleaner/better way of doing this?